providers/base/units/stress/test-plan.pxu

```diff
@@ -300,7 +300,7 @@ unit: test plan
 _name: Automated iperf3 tests
 _description: Automated iperf3 performance test
 include:
-    ethernet/iperf3_.*
```

Comment:
The original regex matches not only `iperf3_interface` but also subsets with longer segments, such as  `iperf3_reverse_interface`. This results in missing jobs after switching to template-id. 
Regex like this one need to be matched with more than one template